### PR TITLE
Changes item registry names

### PR DESCRIPTION
**Pull request by tyra314**
Registry names have to be in lower case, so it's change to snail_case.
This also adds remappings for the old names, just in case.

### CODE REVIEW

**Comment by tyra314** [comment]
Maybe to add some words, I see two discussion points here:
- Using snail_case or simply lowercase for the registry name
- Is the remapping useful, given that the 1.11.2 branch hasn't seen a release yet? Was it possible in the past, to upgrade a map with only AntiqueAtlas from one MC version to the next? (Though I haven't checked yet, if it would work to upgrade from for example 1.10.2 with this)

**Comment by Hunternif** [comment]
The last release was for 1.9.4, and that it where registry was introduced, correct?
In that case we should include a migration mechanism to allow people to keep their old atlases.

**Comment by tyra314** [comment]
So I did the test. Created a new World in singleplayer with only 1.9.4, forge and the version of antique atlas available on curse. Imported the world into my dev env and it worked 😀

**Review by osipxd — CHANGES_REQUESTED** [review]
I think better to use String#equalsIgnoreCase() when you check missed mappings

File: src/main/java/hunternif/mc/atlas/AntiqueAtlasMod.java [hunk 1]
```diff
@@ -116,4 +117,20 @@ public void init(FMLInitializationEvent event){
...
 		proxy.postInit(event);
 	}
+
+	@Mod.EventHandler
+	public void onMissingMapping(FMLMissingMappingsEvent event) {
+		Log.info("Repairing missing mappings");
+		for (FMLMissingMappingsEvent.MissingMapping mapping : event.get()) {
+			String resourcePath = mapping.resourceLocation.getResourcePath().toLowerCase();
+			if (mapping.type == GameRegistry.Type.ITEM) {
+				if ("antiqueatlas".equals(resourcePath)) {
```
Comment (osipxd) [inline src/main/java/hunternif/mc/atlas/AntiqueAtlasMod.java]: Here

File: src/main/java/hunternif/mc/atlas/AntiqueAtlasMod.java [hunk 2]
```diff
@@ -116,4 +117,20 @@ public void init(FMLInitializationEvent event){
...
+					mapping.remap(itemAtlas);
+				} else if ("emptyantiqueatlas".equals(resourcePath)) {
```
Comment (osipxd) [inline src/main/java/hunternif/mc/atlas/AntiqueAtlasMod.java]: And here

**Review by osipxd — APPROVED** [review]

**Pull request merged**